frontend/frontend-auto-translate.php

```diff
@@ -275,7 +275,7 @@ protected function get_translated_term_by( $field, $term, $taxonomy ) {
 				return $tr_id;
 			}
 		} else {
-			$terms = get_terms( $taxonomy, array( $field => $term, 'lang' => '' ) );
+			$terms = get_terms( array( 'taxonomy' => $taxonomy, $field => $term, 'lang' => '' ) );
 
 			if ( ! empty( $terms ) && ! is_wp_error( $terms ) ) {
 				$t = reset( $terms );
```

Comment:
```
 ------ -------------------------------------------------------------
  Line   frontend/frontend-auto-translate.php
 ------ -------------------------------------------------------------
  281    Parameter #1 $array of function reset expects array|object,
         array<int, int|string|WP_Term>|string given.
 ------ -------------------------------------------------------------
```

I think this can be fixed by replacing `&& ! is_wp_error( $terms )` by `&& is_array( $terms )` at L280:
```suggestion
			$terms = get_terms( array( 'taxonomy' => $taxonomy, $field => $term, 'lang' => '' ) );

			if ( ! empty( $terms ) && is_array( $terms ) ) {
				$t = reset( $terms );
```

Reply:
I'll look at this. I thought about something like that. Thank you for your confirmation / suggestion.

Reply:
It seems not to be sufficient. I complete with another condition on the `reset()` result to be sure we can do `$t->term_id`

Reply:
Ok, because `get_terms()` can return `string[]`.